setup.py

```diff
@@ -13,7 +13,14 @@
       author_email='archive@rockarch.org',
       license='MIT',
       packages=find_packages(),
-      install_requires=[],
+      install_requires=["ArchivesSnake>=0.8.1",
+                        "boltons>=20.0.0",
+                        "fuzzywuzzy>=0.17.0",
+                        "pre-commit>=1.18.3"
+                        "pytest",
+                        "PyYAML>=5.2.0",
+                        "sphinx>=1.8.5",
+                        "tox>=3.14.0"],
       tests_require=[],
```

Comment:
```suggestion
      tests_require=["pytest", "pre-commit", "tox", "sphinx"],
```

Reply:
Ohhh, ok. Those requirements make a little more sense to me now.